app/public/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Why are we moving manifest?

Reply:
This is to support [Gnosis App](https://gnosis.io/) as it looks for manifest.json file at the root, if you think you don't need to support Gnosis, it can be reverted.

https://docs.gnosis-safe.io/build/sdks/safe-apps/releasing-your-safe-app